### ISSUE TITLE
cantSplit doesn't work with a rowSpan

### ISSUE THREAD

**Issue by kaddaric** [dolanmiu/docx#541]
In a table, the 'cantSplit' is not working when it's a rowSpan

**Comment by pongstylin** [comment]
I don't believe it is supposed to.  From a Word XML perspective, cantSplit is applied to the actual rows.  Just because you use a rowSpan on a cell doesn't change how cantSplit is applied.

What you need is described in #668.    The "keepNext" property must be set on all of the paragraphs in all of the cells in all except the last of the rows covered by the spanned cell.  This will prevent "splitting" the cell with a rowSpan.  Unfortunately, you are only able to achieve this on docx 4.x.  It doesn't seem to be supported by docx 5.x.
